gapis/api/gles/replay.go

```diff
@@ -234,6 +248,11 @@ func (a API) Replay(
 		transforms.Add(rf)
 	}
 
+	if profile != nil {
+		// Don't use DCE. TODO: should we?
```

Comment:
I don't think we should be using DCE for profiling. It would be way too easy for it to give completely wrong answers.